docs/stdlib/set.rst

```diff
@@ -348,15 +352,18 @@ Sets
 
     :index: multiplicity uniqueness
 
-    Check that the input set contains only unique elements, i.e a *proper set*.
+
+    Returns :eql:type:`anytype` based on condition of if the set is distinct.
+
+    This checks the input set if it contains only unique elements, i.e a
+    *proper set*.
 
     If the input set contains duplicate elements, ``assert_distinct`` raises a
-    ``ConstraintViolationError``.  This function is useful
-    as a runtime distinctness assertion in queries and computed
-    expressions that should always return proper sets, but where static
-    multiplicity inference is not capable enough or outright impossible.
-    An optional *message* named argument can be used to customize the error
-    message.
+    ``ConstraintViolationError``. This function is useful as a runtime
+    distinctness assertion in queries and computed expressions that should
+    always return proper sets, but where static multiplicity inference is not
+    capable enough or outright impossible. An optional ``message`` named
```

Comment:
```suggestion
    capable enough or outright impossible. An optional *message* named
```
Generally, we format references to *parameter names* differently from actual language `expressions`.